lib/node_modules/@stdlib/array/base/broadcasted-quinary4d/package.json

```diff
@@ -0,0 +1,67 @@
+{
+    "name": "@stdlib/array/base/broadcasted-quinary4d",
```

Comment:
This file has incorrect indentation. Should be 2-space, hence the CI lint error.